README.md

```diff
@@ -20,15 +20,23 @@ or the language specifier after them.
 ### Question 1
 
 Define a method called `offer_rose`, which should take one argument named `person`.
+// class Flower
+    def offer_rose
+      return "rose"
+      end
+  end
 
 When called the method should `puts` "Would you take this rose, `person`, in exchange for giving an old beggar woman shelter from the bitter cold?"
+// class Flower
+    def offer_rose
+      puts "Would you take this rose, `person`, in exchange for giving an old beggar woman shelter from the bitter cold?"
+      end
+  end
```

Comment:
This method is closer, but the puts statement should use string interpolation to print out whatever the argument to the method is. So when all is said and done, it should look something like this...

```rb
def offer_rose(person)
  puts "Would you take this rose, #{person}, in exchange for giving an old beggar woman shelter from the bitter cold?"
end
```